src/store/index.js

```diff
@@ -1,6 +1,6 @@
 import { createStore } from "redux";
-import rootReducer from "../reducers/index";
+import rootReducer from "../reducers";
 
-const store = createStore(rootReducer);
+const store = createStore(rootReducer, window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__());
```

Comment:
Probably it's ok, but what is it for?

Reply:
redux extension for chrome that allows monitor global state as well

Reply:
Ok, let it be